test/Libraries/SystemTestServices/SystemTestBase.cs

```diff
@@ -194,7 +194,8 @@ protected virtual void StartDynamo(TestSessionConfiguration testConfig)
                     StartInTestMode = true,
                     PathResolver = pathResolver,
                     GeometryFactoryPath = preloader.GeometryFactoryPath,
-                    ProcessMode = TaskProcessMode.Synchronous
+                    ProcessMode = TaskProcessMode.Synchronous,
+                    DynamoCorePath = testConfig.DynamoCorePath
```

Comment:
Where is `testConfig` defined? Shouldn't it be `DynamoBasePath` as defined in the `TestServices.dll.config` file?

Reply:
This is where the testCOnfig is created
https://github.com/DynamoDS/Dynamo/blob/master/test/Libraries/SystemTestServices/SystemTestBase.cs#L65
`DynamoBasePath` gets assigned to testCOnfig.DynamoBasePath
By default (when the TestServices.dll.config is missing) testCOnfig.DynamoBasePath will point to the ExecutingAssembly.Location.